inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -38,16 +38,32 @@ def acronym(self, key, value):
     self['closing_date'] = value.get('y')
     self['cnum'] = value.get('g')
     self['place'] = value.get('c')
-    self['subtitle'] = value.get('b')
-    self['title'] = value.get('a')
+    self.setdefault('titles', [])
+    titles = {}
+    if 'a' in value:
```

Comment:
@jalavik Should this kind of checking be done elsewhere also? In most cases we are only doing simple getting with a default value of None.